### ISSUE TITLE
[QUERY] Regarding Distribution of CLI apps to other Workspaces.

### ISSUE THREAD

**Issue by pranith12345** [slackapi/deno-slack-sdk#202]
We have developed an app using the Latest CLI framework and we are currently looking the way to deploy this app to other workspaces . This app is using the External OAuth Authentication to Third-party App and using the Access token to access the APIs . 
This is working when we tested and deployed in my local workspace . 
We were looking a way to test this in other Workspaces outside of our environment.

Is the Above app (which uses External OAuth Authentication) can be implemented using the bolt framework

**Comment by pranith12345** [comment]
Yes . We are looking to distribute the app to workspace where we are currently not a member off

**Comment by filmaj** [comment]
At this time, it's not possible to distribute automation apps to workspaces you are not a member of or do not have access to, but follow [our changelog](https://api.slack.com/automation/changelog) for future announcements around this.

**Comment by tjhiggins** [comment]
@filmaj To confirm this sdk doesn't yet support an install link/oauth? Will keep watching the changelog, but any updates?

**Comment by filmaj** [comment]
@tjhiggins correct, no support for this yet, but it's on our roadmap. I think the only 'workaround' we have so far is to open source your application and let others install it into their own workspaces using the CLI - but this is not an option for many. Sorry I don't have a better answer right now! I hope we will in the coming months, though.
